hack/swift/Makefile

```diff
@@ -87,6 +91,18 @@ byocni-up: rg-up net-up ## Brings up a swift cluster $name in $SUB/$REGION
 		--yes
 	@$(MAKE) set-kubeconf
 
+overlay-up: rg-up overlay-net-up ## Brings up an overlay cluster $name in $SUB/$REGION
+	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
+		--max-pods 250 \
+		--node-count 3 \
+		--network-plugin azure \
+		--network-plugin-mode overlay \
```

Comment:
Does this work? I think we will end up with the AKS managed CNS and AzCNI on the nodes. 
We were able to use prod AKS clusters for SWIFT because I made sure BYO CNI supported SWIFT from the back-end when a podsubnet was set ("no CNI but SWIFT-capable"), but here I don't think we have a way to indicate "no CNI but overlay-capable"
cc @thatmattlong is BYO+Overlay on something you have considered? Will we need to build an AKS standalone to run Overlay E2Es in, again?